frontend/apps/filemanager/filemanager.lua

```diff
@@ -377,8 +378,11 @@ function FileChooser:onBack()
     elseif back_to_exit == "no" then
         return true
     elseif back_to_exit == "prompt" then
-        --TODO
-        print("prompt user")
+        Trapper:wrap(function()
```

Comment:
Trapper huh? I think a regular old ConfirmBox should do. ;-)

http://koreader.rocks/doc/modules/ui.widget.confirmbox.html

Reply:
Yeah trapper...
I don't know, I was watching a poire-z commit and it looked simple.

But yes, better with ConfirmBox

Reply:
Trapper would have been a bit overkill for that indeed :)
It's a bit of black magic with coroutines, and is useful when you have some dialog driven stuff with multiple questions, to replace a bunch of imbricated
```lua
UIManager:show(Confirmbox:new{
 ok_callback = function()
  UIManager:show(Confirmbox:new{
   ok_callback = function()
    UIManager:show(Confirmbox:new{
     ok_callback = function()
      UIManager:show(Confirmbox:new{
       ok_callback = ...
```
with a more linear/logical workflow.